packages/client/src/grant-token.ts

```diff
@@ -32,8 +32,7 @@ export const grantTokenByAuthorizationCode = async (
   const result = TokenSetParametersSchema.safeParse(response.data);
 
   if (!result.success) {
-    // Will get a lint error without `as ZodError`, weird
-    throw new OPError({ originalError: result.error as ZodError });
+    throw new OPError({ originalError: result.error });
```

Comment:
let's add a linear issue of renaming to `LogtoError`

Reply:
LOG-176